ddtrace/contrib/pytest/plugin.py

```diff
@@ -83,6 +84,14 @@ def pytest_configure(config):
         Pin(tags=ci_tags, _config=ddtrace.config.pytest).onto(config)
 
 
+def pytest_sessionstart(session):
+    pin = Pin.get_from(session.config)
+    if pin is not None:
+        filters = pin.tracer._filters
+        filters += [TraceCiVisibilityFilter()]
```

Comment:
Can we make sure to check if this filter is already in the list of filters first so we don't append multiple times?